lib/slack.rb

```diff
@@ -6,6 +6,18 @@ def post_deployers_for_today(deployers)
       post(text: "Today’s deployer is *#{deployers[0]}*. Reserves: *#{deployers[1]}*, *#{deployers[2]}*")
     end
 
+    def post_confused_features
+      return unless Date.today.on_weekday?
+
+      confused_features = Features.new.all.select { |f| f.state == 'confused' }
```

Comment:
What do you think about passing these in instead of querying them in here? So this class can stay focussed on formatting and sending messages

Reply:
I get the idea, but I think having everything in the same method also has its advantages here. Maybe a good refactor if this class grows more?